packages/rx_bloc_cli/mason_templates/bricks/lib_auth/__brick__/bin/server/controllers/authentication_controller.dart

```diff
@@ -56,6 +64,18 @@ class AuthenticationController extends ApiController {
     final token = _authenticationService.issueNewToken(null);
     return responseBuilder.buildOK(data: token.toJson());
   }
+{{#enable_apple_auth}}
+  Future<Response> _authenticateWithAppleHandler(Request request) async {
+    final params = await request.bodyFromFormData();
+
+    throwIfEmpty(
+      params['authorizationCode'],
+      BadRequestException('The user authorization with Apple felt.'),
```

Comment:
The exception message should probably be "The user authorization with Apple failed"

Reply:
Edited